docs/test-start-server.md

```diff
@@ -1,44 +1,38 @@
 #### test-start-server
 ---
-Allows test suite to skip the start and stop of the server via pom configuration or command line arguments.
+Start a Liberty Profile server in background. The server instance will be automatically created if it does not exist. test-start-server honors the skipTests, skipITs, and maven-test-skip properties. It also allows you to bypass automatically starting the server during the pre-integration-test phase with pom configuration or a Liberty-specific command line argument.
```

Comment:
Remove the word "Profile" - sorry about that.   The history is that we used to call it Liberty Profile, but we don't any longer.  I let that slip in.

Also add `ticks` to the keywords like `test-start-server`, `skipTests`, etc.